lua/plenary/path.lua

```diff
@@ -198,6 +198,57 @@ function Path:expand()
   return expanded and expanded or error("Path not valid")
 end
 
+function Path:make_relative()
+  if self.filename:sub(1, #self._cwd) == self._cwd  then
```

Comment:
You might have misunderstood me here or its my english (probably latter).
I thought it might be cool that if cwd is null we fallback to `self._cwd` because we actually still wanna pass in `cwd`.

E.g. We are in `~/repos/telescope.nvim` and have a mapping: `lua require'telescope.builtin'.find_files({ cwd = '~/.config/nvim' })`. `self._cwd` would be `~/repos/telescope.nvim` in this occasion so we still need to pass in cwd.

You can use `cwd = F.if_nil(cwd, self._cwd, cwd)` to realize that

Does that make sense?

Reply:
Gotcha yeah that makes sense. I'll push that change soon